custom/lib_portal/models/issue_book.py

```diff
@@ -0,0 +1,44 @@
+
+from odoo import fields, models
+from datetime import datetime, timedelta
+
+class IssueBook(models.Model):
+    _name = "issue.book"
+    _description = "Library Book Lending Mangement"
+
+
+    fromDate = fields.Date('Start Date',default=datetime.today())
+    toDate = fields.Date(string="Due Date",default=datetime.today()+timedelta(days=30))
```

Comment:
* for the default values of date fields it is recommended to use 'default=fields.Date.context_today' this will save you problems related to timezone differences.

Reply:
Got it.Changed the code accordingly.Will commit it